### PR TITLE
timezonemap: init at 0.4.5

### PR DESCRIPTION
**Pull request by mkg20001**
###### Motivation for this change

Needed for cinnamon

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/timezonemap/default.nix [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{ stdenv
+, autoreconfHook
+, fetchbzr
+, pkgconfig
+, gtk3
+, glib
+, file
+, gobject-introspection
+, json_glib
```
Comment (cole-h) [inline pkgs/development/libraries/timezonemap/default.nix:9]:
```suggestion
, json-glib
```

`json_glib` is an alias to `json-glib`.

File: pkgs/development/libraries/timezonemap/default.nix [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  nativeBuildInputs = [
+    pkgconfig
+    autoreconfHook
+  ];
+
+  buildInputs = [
+    gtk3
+    glib
+    gobject-introspection
+    json_glib
```
Comment (cole-h) [inline pkgs/development/libraries/timezonemap/default.nix:32]:
```suggestion
    json-glib
```

Missed this one.

File: pkgs/development/libraries/timezonemap/default.nix [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  src = fetchbzr {
+    url = "lp:timezonemap";
+    rev = "58";
+    sha256 = "1qdp5f9zd8c02bf0mq4w15rlhz2g51phml5qg9asdyfd1715f8n0";
+  };
+
+  nativeBuildInputs = [
+    pkgconfig
+    autoreconfHook
```
Comment (worldofpeace) [inline pkgs/development/libraries/timezonemap/default.nix:25]:
```suggestion
    autoreconfHook
    wrapGAppsHook
```

File: pkgs/development/libraries/timezonemap/default.nix [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  ];
+
+  configureFlags = [
+    "CFLAGS=-Wno-error"
+    "--sysconfdir=/etc"
+    "--localstatedir=/var"
+  ];
+
+  installFlags = [
+    "sysconfdir=\${out}/etc"
```
Comment (worldofpeace) [inline pkgs/development/libraries/timezonemap/default.nix:43]:
```suggestion
    "sysconfdir=${placehoulder "out"}/etc"
```

**Comment by mkg20001** [comment]
merge?

**Comment by worldofpeace** [comment]
thanks @mkg20001

**Pull request merged**